src/client/src/Domains/Student/StudentProfileForm/index.tsx

```diff
@@ -29,7 +30,9 @@ export interface IStudentProfileForm {
     classStanding: string; 
     email: string; 
     biography: string;
-}
+    resume: string;
```

Comment:
I think these two should be of type File. Also they should be optional (i.e. transcript?: File)